pkg/controller/servicebindingrequest/binder_test.go

```diff
@@ -145,9 +147,10 @@ func TestAppendEnvVar(t *testing.T) {
 func TestBinderApplicationName(t *testing.T) {
 	ns := "binder"
 	name := "service-binding-request"
+	applicationGVR := schema.GroupVersionResource{Group: "apps", Version: "v1", Resource: "deployments"}
```

Comment:
```suggestion
	applicationGVR := appsv1.SchemeGroupVersion.WithResource("deployments")
```
I wonder whether "deployments" isn't defined as a const somewhere as well.

Reply:
Addressed 
c277bf9